syntax = "proto3";

package transeTest;

message testBody_b {
	uint32 id = 1;
	double double_test = 2;
}
